contracts.yml

```diff
@@ -0,0 +1,38 @@
+handle: external-event
```

Comment:
@codewithcheese What happens when this type already exists in Jellyfish?

Reply:
@joshbwlng for now there is not check, but we will brainstorm a way to reference and existing type or to denote that the definition is a reference to an exisiting type